### ISSUE TITLE
Syntax highlighting thrown off by double quote between single quotes

### ISSUE THREAD

**Issue by Ruin0x11** [Emacs-Kotlin-Mode-Maintainers/kotlin-mode#28]
In Kotlin a pair of single quotes denotes a character, but there is no handling for the case of a double quote inside the single quotes, leading to this:

![image](http://sub.god.jp/f/npbtey.png)

Here is the code.

```kotlin
fun test() {
    val char = myChar()
    if (char == '"') {
        println("Got double quote.")
    }
}
```

**Comment by Spajderix** [comment]
Same thing happens for multiline strings and a double quote inside.

**Comment by gregghz** [comment]
This was partially fixed by #42 but the multiline string portion is not yet resolved.
